sanity/doctors/ParcelDoctor.js

```diff
@@ -75,10 +92,10 @@ export class ParcelDoctor extends Doctor {
     return !!currentOwner && parcel.owner !== currentOwner
   }
 
-  isUpdateOperatorMismatch(currentUpdateOperator, parcel) {
+  isOperatorMismatch(currentOperator, operator) {
     return (
-      !!currentUpdateOperator &&
-      parcel.update_operator !== currentUpdateOperator
+      (!operator && !Contract.isEmptyAddress(currentOperator)) ||
+      (operator && operator !== currentOperator)
```

Comment:
This seems to be the same method as `isOperatorMismatch` on EstateDoctor, I think we might need to avoid duplication here.

Some ideas:
- Have an `Address` class:
```javascript
new Address(currentOperator).equals(operator)
// or
new EthAddress(currentOperator).equals(operator)
```
which would also allow for some other abstractions regarding addresses
- The same can be accomplished by having a file with functions. A bit worse for autocompletion and discoverability but maybe less complex.
- We could use composition

```javascript
export class ParcelDoctor extends Doctor {
  constructor(...props) {
    super(...props)
    this.assetDoctor = new AssetDoctor(this)
  }

  method() {
    this.assetDoctor.isAddressMismatch()
    // or
    !this.assetDoctor.isSameAddress()
  }
}
// (...)

export class EstateDoctor extends Doctor {
  constructor(...props) {
    super(...props)
    this.assetDoctor = new AssetDoctor(this)
  }

  method() {
    this.assetDoctor.isAddressMismatch()
    // or
    !this.assetDoctor.isSameAddress()
  }
}
```

we can maybe reuse the `ethereum/` folder for the first two

Reply:
I like the idea but I think that it should be part of another PR. 

What do you think? I can create an issue

Reply:
🆗